cms/static/sass/_variables.scss

```diff
@@ -1,7 +1,7 @@
 // studio - utilities - variables
 // ====================
 
-// Table of Contents 
```

Comment:
Sorry for the annoyance here. My Sublime settings strips ending whitespace, so this update also includes a bunch of this automatic cleanup. Feel free to ignore.